documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,161 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
```

Comment:
and add OVA after OSP

Reply:
@ahadas FYI, the order of the ifdef statements in adding-source-provider.doc does not affect the order they appear. The order they appear in is determined by what is in the master.adoc file.  I thought we agreed that we would follow the order of the providers in the UI -- and when the order in the UI changes, the order in the documentation will change. Would you prefer me to change the order now?

Reply:
I thought we already changed the order in the UI but I see we didn't - wrote to @yaacov on Jira about it
but yeah, to your question - I think it's better to have a reasonable order in the documentation rather than following the order of the items in the UI, in case those two are different

Reply:
https://github.com/kubev2v/forklift-console-plugin/pull/698 - merged upstream